core/server/middleware/serve-public-file.js

```diff
@@ -4,18 +4,18 @@ var crypto = require('crypto'),
     config = require('../config'),
     utils  = require('../utils');
 
-// ### ServeSharedFile Middleware
+// ### servePublicFile Middleware
 // Handles requests to robots.txt and favicon.ico (and caches them)
-function serveSharedFile(file, type, maxAge) {
+function servePublicFile(file, type, maxAge) {
     var content,
-        corePath = config.get('paths').corePath,
+        publicFilePath = config.get('paths').publicFilePath,
         filePath,
         blogRegex = /(\{\{blog-url\}\})/g,
         apiRegex = /(\{\{api-url\}\})/g;
 
-    filePath = file.match(/^shared/) ? path.join(corePath, file) : path.join(corePath, 'shared', file);
+    filePath = file.match(/^public/) ? path.join(publicFilePath, file.replace(/^public/, '')) : path.join(publicFilePath, file);
```

Comment:
Let's remove this line, and also the `public/` from the paths in blog/app.js and make this consistent? All the assets are now neatly colocated, let's keep the code clean?